include/git2/config.h

```diff
@@ -398,23 +398,31 @@ GIT_EXTERN(int) git_config_get_string_buf(git_buf *out, const git_config *cfg, c
  *
  * The callback will be called on each variable found
  *
+ * The regular expression is applied case-sensitively on the normalized form of
+ * the variable name: the case-insensitive parts are lower-case.
+ *
  * @param cfg where to look for the variable
  * @param name the variable's name
  * @param regexp regular expression to filter which variables we're
  * interested in. Use NULL to indicate all
  * @param callback the function to be called on each value of the variable
  * @param payload opaque pointer to pass to the callback
+ *
```

Comment:
Is there a reason why you've added those empty comment lines?

Reply:
I originally put the extra comment after the params, but then realised the other explanations were before, and my cut-and-paste was evidently insufficient. I'll go clean these up.

Reply:
The extra line is now removed.